vm/page.c

```diff
@@ -196,14 +190,6 @@ page_t *_page_get(addr_t addr)
 }
 
 
-void vm_pageFreeAt(pmap_t *pmap, void *vaddr)
```

Comment:
Why is `vm_pageFreeAt()` removed only from here and not from vm/page-nommu.c or vm/page.h?

Reply:
Because I didn't notice that I didn't add these files to a commit. I've fixed that now.